pkg/flags/global.go

```diff
@@ -23,9 +23,14 @@ import (
 	"github.com/abcxyz/pkg/cli"
 )
 
+const (
+	PlatformTypeLocal  = "local"
```

Comment:
These constants seem to be the same as the similarly-named constants in reporter.go. Do we really need two copies?

Reply:
I think this is a coincidence, but are technically different and may diverge.